cmd/src/campaigns_common.go

```diff
@@ -85,8 +85,8 @@ func newCampaignsApplyFlags(flagSet *flag.FlagSet, cacheDir, tempDir string) *ca
 	flagSet.StringVar(&caf.namespace, "n", "", "Alias for -namespace.")
 
 	flagSet.IntVar(
-		&caf.parallelism, "j", 0,
-		"The maximum number of parallel jobs. (Default: GOMAXPROCS.)",
+		&caf.parallelism, "j", runtime.GOMAXPROCS(0),
+		"The maximum number of parallel jobs. Default is GOMAXPROCS.",
```

Comment:
Note for reviewers: this shoudln't be strictly part of this PR, but I wanted to clean up something.